lib/net.c

```diff
@@ -169,6 +190,86 @@ get_if_back(struct net_config *net_conf)
 	return &net_conf->back;
 }
 
+int
+gatekeeper_setup_rss(uint8_t portid, uint16_t queues[], uint16_t num_queues)
```

Comment:
Please change `uint16_t queues[]` to `uint16_t *queues`. The alternative notation used can lead to troubles.